editor/src/viewport_tools/tool_message.rs

```diff
@@ -25,6 +25,9 @@ pub enum ToolMessage {
 	Fill(FillMessage),
 	#[remain::unsorted]
 	#[child]
+	Freehand(FreehandMessage),
+	#[remain::unsorted]
+	#[child]
```

Comment:
Please put below `Pen`. Actually some others of these are in the wrong order. Would you mind rearranging them in the order that they appear in the tool shelf of the app?